source/guides/references/error-messages.md

```diff
@@ -535,6 +534,29 @@ This error likely appeared because:
 1. You do not have internet. Please ensure you have connectivity then try again.
 2. You are a developer that has forked our codebase and do not have access to run our API locally. Please read more about this in our {% url "contributing doc" https://on.cypress.io/contributing %}.
 
+## <a id="bad-policy-settings"></a> {% fa fa-exclamation-triangle red %} Cypress detected policy settings on your computer that may cause issues
```

Comment:
Does this `<a>` anchor link even work? The anchor link is created from the Markdown itself - so this shouldn't be necessary. Similar to how GitHub creates anchor links at each # in markdown. Hash should automatically be `#Cypress-detected-policy-settings-on-your-computer-that-may-cause-issues` - then you need to set up a link in `on.cypress.io` to redirect to this url + hash.

Reply:
Yeah, it does work, I wanted to have a shorter anchor link and figured this would be the way.

You're saying add something like `https://on.cypress.io/bad-policy-warning -> https://docs.cypress.io/.../error-messages.html#Cypress-detected-policy-settings-on-your-computer-that-may-cause-issues`?